pytype/metrics.py

```diff
@@ -336,6 +340,8 @@ class Snapshot(Metric):
 
   def __init__(self, name, enabled=False, groupby="lineno",
                nframes=1, count=10):
+    if enabled and tracemalloc is None:
+      raise Exception("tracemalloc module couldn't be imported")
```

Comment:
Could you raise a more specific exception type? (Maybe ImportError?)

Reply:
At this point, I think that raising `ImportError` is not perfect because the import happened before (and was the cause of the error), but here we want to notify that the snapshot metric doesn’t work, which is a different kind of error. But I agree that raising a very general exception type is also not very good. Unless you have another idea, I would be fine with both leaving and changing it.

Reply:
How about `ValueError` or `RuntimeError`? I'm not too fussed about which exception class you pick, as long as it makes some sense, but I'd strongly prefer not to leave it as an entirely uninformative `Exception`.

Reply:
Thanks for the suggestion. I changed it.